src/main/java/cat/udl/eps/softarch/demo/domain/Business.java

```diff
@@ -0,0 +1,69 @@
+package cat.udl.eps.softarch.demo.domain;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Entity
+@Table(name = "business")
+@Data
+@EqualsAndHashCode
+public class Business {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotBlank
+    @Column(nullable = false)
+    private String address;
+
+    @NotBlank
+    @Column(nullable = false)
+    private String name;
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private BusinessStatus status;
+
+    public Business() {}
```

Comment:
You can omit all constructors, getters and setters. With the Lombok all these are automatically generated